server/src/main/resources/org/jboss/as/server/controller/descriptions/LocalDescriptions.properties

```diff
@@ -257,7 +257,7 @@ server-set-restart-required=Puts the server into a restart-required mode.
 write-config=An operation to force the server to write its config file, without making any actual config change.
 
 # Deployments
-deployment=A deployment represents anything that can be deployed (e.g. an application such as EJB-JAR, WAR, EAR, any kind of standard archive such as RAR or JBoss-specific deployment) into a server.
+deployment=A deployment represents anything that can be deployed (e.g. an application such as Jakarta EJB-JAR, WAR, EAR, any kind of standard archive such as RAR or JBoss-specific deployment) into a server.
```

Comment:
I'm not 100% sure if that is the correct one "EJB-JAR" vs "Jakarta EJB-JAR"

Maybe "Jakarta Enterprise Beans-JAR" instead?

Reply:
`Jakarta Enterprise Beans-JAR` will be a better choice I think.

Reply:
I agree, "Jakarta Enterprise Beans-JAR" seems good.